Include/cpython/fileutils.h

```diff
@@ -32,6 +32,12 @@ PyAPI_FUNC(int) _Py_EncodeLocaleEx(
     int current_locale,
     _Py_error_handler errors);
 
+#ifdef HAVE_NON_UNICODE_WCHAR_T_REPRESENTATION
+PyAPI_FUNC(char32_t*) _Py_convert_wchar_t_to_UTF32(
```

Comment:
In the latest Unicode version, only 0x10ffff characters are used. We are very far from needing UTF-32 where code points are encoded as 2 units (64 bits). IMO it's more honest to say "UCS4" than "UTF32" here. Also, it makes more sense in PyUnicode_FromWideChar() which calls _PyUnicode_FromUCS4().

Reply:
I see, that makes sense.